pkg/operator/status.go

```diff
@@ -55,15 +61,20 @@ func (optr *Operator) syncProgressingStatus() error {
 	}
 
 	optrVersion, _ := optr.vStore.Get("operator")
-	var message string
+	status := configv1.ConditionFalse
+	message := fmt.Sprintf("Cluster version is %s", optrVersion)
+
 	if optr.vStore.Equal(co.Status.Versions) {
-		// syncing the state to existing version.
-		message = fmt.Sprintf("Running resync for %s", optrVersion)
+		if optr.inClusterBringup {
+			status = configv1.ConditionTrue
+		}
```

Comment:
Wait why would we be `available = false` if the version store is equal and `!inClusterBringup`?

Reply:
this is setting Progressing to false, not available. If `!inClusterBringup && version-equal`, `available is gonna be true (assuming failing is false as well). Am I missing something?

Reply:
Sorry, yeah.  I guess it's clearer when viewing the surrounding code as the functions clearly say `syncProgressingStatus` and `syncAvailableStatus`.

So nevermind, LGTM!